cms/djangoapps/contentstore/tasks.py

```diff
@@ -58,7 +58,7 @@ def rerun_course(source_course_key_string, destination_course_key_string, user_i
             # it's possible there was an error even before the course module was created
             pass
 
-        return "exception: " + unicode(exc)
+        return ("exception: " + unicode(exc))[:999]  # don't exceed 1000 chars
```

Comment:
@dan-f yes, SQL thing, see the above gist.

TBH this is kind of a band-aid fix in my view.  If we intend to store entire tracebacks in a database column that column ought to be provisioned with enough space to accomodate one, otherwise we should log it to a file where it's going to be more reasonably accessible via better tools for the job (i.e. splunk).

my main concern with this PR is that we might have failures creating reruns and lose any info we could use to diagnose those failures.